tests/core/strip/strip_test.go

```diff
@@ -259,10 +306,35 @@ var wantStackTrace = []string{
 	`^$`,
 }
 
-func embedWantedStackTraces() string {
+func embedWantedTestStackTrace() string {
+	return embedStringSliceVar("wantTestStackTrace", wantTestStackTrace)
+}
+
+var wantTestStackTrace = []string{
+	`^goroutine \d+ \[running\]:$`,
```

Comment:
This is very comprehensive, but also somewhat prone to breaking with future versions of Go. Could you pick out one or two lines that are essential for verifying the presence of debug information and drop the rest?

Reply:
Thanks for the review! I changed the testing so that it reuses exactly the previous expectations and doesn't look at the tail that's due to the implementation of go's testing package. See if this pleases.